### PR TITLE
Update rollkit app for wordle.md

### PR DESCRIPTION
**Pull request by toschdev**
Updates the rollkit app to v0.2.1

## Summary by CodeRabbit

- **Documentation**
  - Updated the tutorial to reflect the new Rollkit app version (`v0.2.1`) for installation via Ignite.

### CODE REVIEW

File: tutorials/wordle.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ Rollkit on our codebase.
 To install the Rollkit app to Ignite, run the following command:
 
 ```bash
-ignite app install github.com/ignite/apps/rollkit@rollkit/v0.2.0
+ignite app install github.com/ignite/apps/rollkit@rollkit/v0.2.1
```
Comment (jcstein) [inline tutorials/wordle.md:172]: IIUC @yarikbratashchuk - this could use the variable instead, right?

**Review by jcstein — CHANGES_REQUESTED** [review]
cc @yarikbratashchuk

File: tutorials/wordle.md (hunk 1 above)
Comment (yarikbratashchuk) [reply tutorials/wordle.md:172]: yes, i'll make a pr with it and will set it to v0.2.1

**Comment by yarikbratashchuk** [comment]
closing this one, #399 covers it and few other places with versions

**Pull request closed without merging**